src/gwc/src/main/java/org/geoserver/gwc/controller/GwcUrlHandlerMapping.java

```diff
@@ -94,8 +98,8 @@ private static final class Wrapper extends HttpServletRequestWrapper {
 
             // Adjust LOOKUP_PATH used by spring to remove workspace
```

Comment:
Comment still refers to the old constant name.

Reply:
Done